pkg/builtinchecks/yamls/upgrade-strategy.yaml

```diff
@@ -0,0 +1,10 @@
+name: "upgrade-strategy"
```

Comment:
Should we make the name more specific, like "no-rolling-updates" or something. Also I think we should generally use "update" everywhere (names and text) instead of a mix of "upgrade" and "update".

Reply:
Yes, I'll change the check name.  I originally had it enforcing more, but there just isn't going to be a sane set of defaults for anything other than to push using RollingUpdate.

Reply:
Done